### PR TITLE
[MINI-3128] Updated Mini app error message for access token

### PR DESCRIPTION
**Pull request by rleojoseph**
# Description
* Updated error message format for getAccessToken method
* The same error should be followed for any new methods that is added to the SDK

# Checklist
- [x] I have read the [contributing guidelines](CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `fastlane ci` without errors

### CODE REVIEW

**Review by corycaywood — APPROVED** [review]
LGTM 👍

File: MiniApp/Classes/core/Utilities/MASDKLocale.swift [hunk 1]
```diff
@@ -7,7 +7,6 @@ public struct MASDKLocale {
         case cancel                                 = "miniapp.sdk.ios.alert.title.cancel"
         case allow                                  = "miniapp.sdk.ios.ui.allow"
         case save                                   = "miniapp.sdk.all.ui.save"
-        case customSettingsFooter                   = "miniapp.sdk.ios.customsettingsfooter"
```
Comment (Climbatize) [inline MiniApp/Classes/core/Utilities/MASDKLocale.swift:10]: why this deletion?

**Review by Climbatize — APPROVED** [review]
LGTM

File: MiniApp/Classes/core/Utilities/MASDKLocale.swift (hunk 1 above)
Comment (rleojoseph) [reply MiniApp/Classes/core/Utilities/MASDKLocale.swift:10]: Because it's not anywhere

**Pull request merged**